src/web/core/src/main/java/org/geoserver/web/CapabilitiesHomePagePanel.java

```diff
@@ -34,15 +34,19 @@ public class CapabilitiesHomePagePanel extends Panel {
      * model object to this panel's ListView.
      */
     public static class CapsInfo implements Serializable {
-        private static final long serialVersionUID = 1L;
 
+        /**
+         * Service name.
+         */
         String service;
 
+        /** Service version */
         Version version;
 
+        /** GetCapabilities service description */
         String capsLink;
 
-        public CapsInfo(String service, Version version, String capsLink) {
+        public CapsInfo(String service, Version version, String capsLink){
```

Comment:
I guess this is going to get reformatted.

Reply:
yes it was still to be formatted